### DISCUSSION TITLE
Please use squash and merge when merging PRs

### DISCUSSION THREAD

**Discussion by rdsedmundo** [serverless-heaven/serverless-webpack#1547] [Team Posts]
This will make the history a little bit cleaner and easier to revert if something goes wrong.

@HyperBrain it would be good if you could change this in the project settings and force that option to be used.

**Comment by miguel-a-calles-mba** [comment]
@rdsedmundo, I think it is already enabled. We just need to remember to squash merge or have it be the only merge option so we do not forget.

![image](https://user-images.githubusercontent.com/44813512/81828204-64a6c600-94ee-11ea-9166-cd879ce5ab27.png)

**Comment by miguel-a-calles-mba** [comment]
I personally prefer squash merges too.

**Comment by HyperBrain** [comment]
I will change the setting soon. For PR merges it should be ok - for branch
merges it is imo NOT because it swallows PRs on a branch
